### PR TITLE
Add nginx with rtmp module installation script 

### PR DESCRIPTION
**Pull request by melvinheks**
Installation ran with Ubuntu 18.04

### CODE REVIEW

File: scripts/install [hunk 1]
```diff
@@ -22,6 +22,10 @@ else
       source ./lib/install_kafka.sh
       install_kafka
       ;;
+    nginx-rtmp)
+      source ./lib/install_nginx_rtmp.sh
+      install_nginx_with_rtmp
```
Comment (yliu0715) [inline scripts/install]: Don't you also want to add these two lines to the if statement above?

File: scripts/lib/install_nginx_rtmp.sh [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                video/mp2t ts;
+            }
+
+            root /mnt/;
+        }
+    }
+}
+"
+download_rtmp_module() {
+  git clone https://github.com/sergey-dryabzhinsky/nginx-rtmp-module.git
```
Comment (chc5) [inline scripts/lib/install_nginx_rtmp.sh]: Put this module in the appropriate directory.

File: scripts/lib/install_nginx_rtmp.sh [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+}
+
+install_nginx_dependencies() {
+  sudo apt install -y build-essential libpcre3 libpcre3-dev libssl-dev zlib1g zlib1g-dev
+}
+
+download_extract_nginx() {
+  wget http://nginx.org/download/${NGINX_DIRECTORY}.tar.gz
```
Comment (chc5) [inline scripts/lib/install_nginx_rtmp.sh]: Save this tar file in /tmp

File: scripts/lib/install_nginx_rtmp.sh [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  tar -xf ${NGINX_DIRECTORY}.tar.gz
```
Comment (chc5) [inline scripts/lib/install_nginx_rtmp.sh]: Define where this file is going to be saved.

File: scripts/lib/install_nginx_rtmp.sh (hunk 2 above)
Comment (melvinheks) [reply scripts/lib/install_nginx_rtmp.sh]: Placed in /usr/lib/src/

File: scripts/lib/install_nginx_rtmp.sh (hunk 4 above)
Comment (melvinheks) [reply scripts/lib/install_nginx_rtmp.sh]: Placed in /usr/local/src/

File: scripts/lib/install_nginx_rtmp.sh (hunk 3 above)
Comment (melvinheks) [reply scripts/lib/install_nginx_rtmp.sh]: Done

File: scripts/install (hunk 1 above)
Comment (melvinheks) [reply scripts/install]: Done

**Pull request merged**